### PR TITLE
Added SASS validation

### PR DESCRIPTION
**Pull request by Coenego**

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

File: Gruntfile.js [hunk 1]
```diff
@@ -215,6 +215,24 @@ module.exports = function(grunt) {
                 }
             }
         },
+        'sass': {
+            'dist': {
+                'options': {
+                    'check': true,
```
Comment (bp323) [inline Gruntfile.js]: What does `check` do?

Comment (Coenego) [inline Gruntfile.js]: This enables the validation of the scss files

**Comment by bp323** [comment]
Verified and merged.

**Pull request merged**